autoendpoint/src/routers/common.rs

```diff
@@ -136,6 +122,15 @@ pub async fn handle_error(
                 error.errno(),
             );
         }
+        RouterError::Fcm(FcmError::Upstream { status, .. }) => incr_error_metric(
+            metrics,
+            platform,
+            app_id,
+            &format!("upstream_{}", status),
```

Comment:
Sorry, one last thing: `status` here is either `FcmErrorResponse::status` or `StatusCode::to_string()`. I know the latter will contain whitespace (e.g. "200 OK") and the former probably does too? Which makes this an invalid metric tag value

Reply:
Sigh, you're right, with the various back & forth and merges `status` is not the same. I'll modify.

Reply:
Wait, no, That is right.

The `status` here is from `Upstream{status: String, ..}` and reflects the returned `FcmErrorResponse.status`, which is a set of enums. There _was_ as similar error, though, and I've corrected it. I've also added some comments because this was a source of confusion.